docs/quick/k8s/submarine-kubeflow.md

```diff
@@ -0,0 +1,112 @@
+<!-- 
+Licensed to the Apache Software Foundation (ASF) under one
+or more contributor license agreements.  See the NOTICE file
+distributed with this work for additional information
+regarding copyright ownership.  The ASF licenses this file
+to you under the Apache License, Version 2.0 (the
+"License"); you may not use this file except in compliance
+with the License.  You may obtain a copy of the License at
+
+  http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing,
+software distributed under the License is distributed on an
+"AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+KIND, either express or implied.  See the License for the
+specific language governing permissions and limitations
+under the License.
+-->
+
+# Submarine with Kubeflow Tutorial
+Submarine with Kubeflow runs on **Linux** and **macOS**. To be able to run Submarine, the requirements are:
+* Java 8.x
+* Kubeflow 0.6+
+* Submarine 0.3.0
+
+## Prepare K8s cluster
+We recommend to use [`kind`](https://kind.sigs.k8s.io/) to create the K8s cluster using Docker contianer "node".
+```
+kind create cluster --image kindest/node:v1.15.6 --name submarine
+```
+
+> NOTICE: Kuebflow 0.7 compatible with kubernetes 1.14 and 1.15
+
+### Deploy Dashboard
+To deploy Dashboard, execute following command:
+```
+kubectl apply -f https://raw.githubusercontent.com/kubernetes/dashboard/v2.0.0-beta8/aio/deploy/recommended.yaml
+```
+
+### Create RBAC
+```
+kubectl create serviceaccount cluster-admin-dashboard-sa
+kubectl create clusterrolebinding dashboard-admin-sa --clusterrole=cluster-admin --serviceaccount=default:dashboard-admin-sa
+```
+
+### Get access token
+```
+kubectl get secrets
+kubectl describe secret dashboard-admin-sa-token-6nhkx
+```
+
+### Running
+Run the following command:
+```
+kubectl proxy
+```
+
+Now access Dashboard at:
+> http://localhost:8001/api/v1/namespaces/kubernetes-dashboard/services/https:kubernetes-dashboard:/proxy/
+
+## Prepare Kubeflow
+
+### Download kfctl
+Download the kfctl v0.7.0 release from the [Kubeflow Releases Page](https://github.com/kubeflow/kubeflow/releases/tag/v0.7.0). More info see [Kubeflow Prepare your environment](https://www.kubeflow.org/docs/started/k8s/kfctl-k8s-istio/#prepare-your-environment).
+
+Run the following command:
+```
+tar -xvf kfctl_v0.7.0_darwin.tar.gz
+mv kfctl_darwin /usr/local/bin/kfctl
+```
+
+### Deploy Kubeflow
```

Comment:
Again, do we really need to deploy kubefow to use the Kubeflow TF operator? This seems too heavy for me.